frontend/config/locales/en.yml

```diff
@@ -640,8 +640,20 @@ en:
     login_success: Login Successful. Redirecting...
     username_placeholder: Username
     password_placeholder: Password
+    confirm_password: Confirm Password
     username_aria_label: User Name
     password_aria_label: User Password
+    lost_password: Forgot Password?
+    reset_password: Reset Password
+    recover_password: Recover Password
```

Comment:
As a second-language English speaker I'm not completely sure why we would distinguish between recovering or resetting a password. IMO recovering means to restore something which isn't done here as the original password cannot be recovered? If it is used rather synonymous maybe we could pick one of the alternatives - probably reset?

Reply:
As a first-language English speaker, I'd tend to agree. 😆 The "recover" terminology is probably superfluous. I'll clean this up.